receiver/hostmetricsreceiver/README.md

```diff
@@ -141,6 +142,46 @@ service:
       receivers: [hostmetrics, hostmetrics/disk]
 ```
 
+### Collecting host metrics from inside a container (Linux only)
+
+On Linux, the host metrics are collected from the `/proc` filesystem. 
```

Comment:
We should amend this to just say that the metrics are collected from system directories on the filesystem or something similar. Some of the scrapers in the host metrics receiver collect from directories other than `/proc`.

Reply:
Yeah, agreed. This could be more accurate. I will amend it.